src/triage/component/catwalk/storage.py

```diff
@@ -39,82 +48,84 @@ def exists(self):
         s3 = s3fs.S3FileSystem()
         return s3.exists(self.path)
 
-    def write(self, obj):
-        s3 = s3fs.S3FileSystem()
-        with s3.open(self.path, 'wb') as f:
-            joblib.dump(obj, f, compress=True)
-
-    def load(self):
-        s3 = s3fs.S3FileSystem()
-        with s3.open(self.path, 'rb') as f:
-            return joblib.load(f)
-
     def delete(self):
         s3 = s3fs.S3FileSystem()
         s3.rm(self.path)
 
+    def open(self, *args, **kwargs):
+        s3 = s3fs.S3FileSystem()
+        return s3.open(self.path, *args, **kwargs)
+
 
 class FSStore(Store):
     def exists(self):
         return os.path.isfile(self.path)
 
-    def write(self, obj):
-        os.makedirs(os.path.dirname(self.path), exist_ok=True)
-        with open(self.path, 'w+b') as f:
-            joblib.dump(obj, f, compress=True)
-
-    def load(self):
-        with open(self.path, 'rb') as f:
-            return joblib.load(f)
-
     def delete(self):
         os.remove(self.path)
 
+    def open(self, *args, **kwargs):
+        os.makedirs(dirname(self.path), exist_ok=True)
+        return open(self.path, *args, **kwargs)
+
 
 class MemoryStore(Store):
     store = None
 
     def exists(self):
         return self.store is not None
 
-    def write(self, obj):
-        self.store = obj
+    def write(self, bytestream):
+        self.store = bytestream
 
     def load(self):
         return self.store
 
     def delete(self):
         self.store = None
 
+    @contextmanager
+    def open(self, *args, **kwargs):
+        with BytesIO() as f:
+            yield f
```

Comment:
This can't just `return BytesIO(*args, **kwargs)`?

Reply:
Just missed this one